app/uk/gov/hmrc/pegaproofofconceptstubs/models/Payload.scala

```diff
@@ -0,0 +1,26 @@
+/*
+ * Copyright 2024 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package uk.gov.hmrc.pegaproofofconceptstubs.models
+
+import play.api.libs.json.{Json, OFormat}
+
+final case class Payload(data: String)
```

Comment:
this isn't that important but when you've got one thing in a case class there's an optimisation you can do to make things more efficient:
```scala
final case class Payload(data: String) extends AnyVal
```
What this will do is essentially represent instances of `Payload` using the underlying value without a wrapper at the bytecode level - again, not super important but nice thing to know if you didn't already